<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Accounts</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>81</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>List Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="listName"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Accounts</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="accTable">
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Neue Zeile</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Neue Spalte</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
